# Part 1

1. Technically, nothing is printed by line 9 as the line of code that calls the function to execute is not there until line 16. If the assumption is the function is called and what is there by line 9, then the output by line 9 is "values added: 20" because **result** is 10+10 = 20, and the console logs out the string "values added" before it.
2. Technically, nothing is printed by line 13 as the line of code that calls the function to execute is not there until line 16. If the assumption is the function is called and what is there by line 13, then the output by line 13 is "values added: 20", then on a newline "final result: 20", because **result** is 10+10 = 20, and the console logs out the string "values added" before it. Since **result** is a **var** variable, it can be accessed outside of the block it was declared in while still in the function, allowing the code on line 13 to run.
3. Technically, nothing is printed by line 9 as the line of code that calls the function to execute is not there until line 16. If the assumption is the function is called and what is there by line 9, then the output by line 9 is "values added: 20" because **result** is 10+10 = 20, and the console logs out the string "values added" before it.
4. Technically, nothing is printed by line 13 as the line of code that calls the function to execute is not there until line 16. If the assumption is the function is called and what is there by line 13, it outputs an error. The **let** declaration of **result** only allows to work in the scope of the block, and since line 13 is outside the scope of **result**'s block, it is an error.
5. Technically, nothing is printed by line 9 as the line of code that calls the function to execute is not there until line 16. If the assumption is the function is called and what is there by line 9, then there is an error by line 9 because you are trying to reassign a value to the **const result**.
6. Technically, nothing is printed by line 9 as the line of code that calls the function to execute is not there until line 16. If the assumption is the function is called and what is there by line 13, nothing runs because there is an error at line 9.